bitrise.yml

```diff
@@ -0,0 +1,172 @@
+---
+format_version: '11'
+default_step_lib_source: https://github.com/bitrise-io/bitrise-steplib.git
+project_type: ios
+workflows:
+  DeployFirebase:
+    description: |
+      The workflow builds and deploys the app in Firebase and Bitrise step.
+
+      For testing the *retry_on_failure* test repetition mode is enabled.
+
+      Next steps:
+      - Set up [Connecting to an Apple service with API key](https://devcenter.bitrise.io/en/accounts/connecting-to-services/connecting-to-an-apple-service-with-api-key.html##).
+      - Or further customise code signing following our [iOS code signing](https://devcenter.bitrise.io/en/code-signing/ios-code-signing.html) guide.
+    steps:
+    - authenticate-host-with-netrc@0:
+        inputs:
+        - username: "$GITHUB_USER_NAME"
+        - password: "$GITHUB_PASSWORD"
+        - host: github.com
+    - git-clone@6: {}
+    - cache-pull@2: {}
+    - cocoapods-install@2: {}
+    - fastlane-match@0:
+        title: Fastlane Match Development
+        inputs:
+        - git_branch: "$FASTLANE_MATCH_GIT_BRANCH"
+        - app_id: "$APP_BUNDLE_ID"
+        - team_id: "$BITRISE_APLLE_DEV_TEAM_ID"
+        - decrypt_password: "$FASTLANE_MATCH_PASSWORD"
+        - git_url: "$FASTLANE_MATCH_GIT_URL"
```

Comment:
I don't think you need this match to build the adhoc for Firebase. Normally the Development certs are for building locally for a real device.

Reply:
I got this [error](https://discuss.bitrise.io/t/fastlane-match-succeeds-but-xcode-couldnt-find-any-ios-app-development-provisioning-profiles-matching/13540) without this step

Reply:
Can you try setting `Automatically manage signing` to Off. 

<img width="567" alt="Screen Shot 2022-10-28 at 15 01 59" src="https://user-images.githubusercontent.com/6356137/198536509-97e44265-348b-4a0a-8e19-2bfaffc50bb2.png">

And set the signing to match Adhoc distribution locally and push the change. 

Or use fastlane to [set automatic signing to off](https://docs.fastlane.tools/actions/automatic_code_signing/) and [update signing](https://docs.fastlane.tools/actions/update_code_signing_settings/).

I'm looking at [this one](https://github.com/nimblehq/dee-money-ios/blob/develop/bitrise.yml) for example, there's only one match run for the release flow.

Reply:
@blyscuit I removed the match step for development, this time it works with the current xcode setup, did'nt have to change signing settings. There may be some other problems previously which triggers the error. Thanks.

![Screenshot 2022-10-28 at 3 58 03 PM](https://user-images.githubusercontent.com/13636465/198547849-0aab605a-9d1c-452e-a306-4ec2d3acd750.png)